aregion.h

```diff
@@ -199,6 +202,7 @@ class ARegion : public AListElem
 		void WriteEconomy(ostream& f, Faction *, int);
 		void WriteExits(ostream& f, ARegionList *pRegs, int *exits_seen);
 		void WriteReport(ostream& f, Faction *fac, int month, ARegionList *pRegions);
+		void write_json_report(json& j, Faction *fac, int month, ARegionList *pRegions);
```

Comment:
Snake case detected :)
I'm not against, but that looks a bit alien around Pascal-cased statements.

Reply:
Yes...
I am highly partial to a style of TypeNames in pascal case, and functions/data elements in snake case and global constants in SCREAM_CASE.
As I rewrite pieces of this, my intent is to move more and more of it to that style, but doing a mass renaming didn't seem to make sense.